### PR TITLE
Newer python.

### PR DESCRIPTION
**Pull request by blockjesper**
Test in Python 3.9, 3.10, 3.11 and 3.12.

### CODE REVIEW

File: docs/requirements.in [hunk 1]
```diff
@@ -4,4 +4,4 @@
 # When updating dependencies here:
 # - Install pip-tools with `pip install pip-tools`
 # - Use `pip-compile requirements.in` to update transitive dependency versions in requirements.txt
```
Comment (oyvindio) [inline docs/requirements.in:6]: Did you run this to update the transitive dependencies?

File: tox.ini [hunk 2]
```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py38,py39
+envlist = py39,py311
```
Comment (oyvindio) [inline tox.ini:2]: It would be ideal to add py310 here also; I suppose if 3.11 works 3.10 should also work

**Review by oyvindio — COMMENTED** [review]
I think python versions need to be added here https://github.com/fiaas/k8s/blob/master/setup.py#L94-L97 to indicate that the library is compatible with these versions, otherwise it may not install cleanly

The list of tox envirionments used in CI need to be updated here https://github.com/fiaas/k8s/blob/master/.semaphore/semaphore.yml#L44 to run tests on the new python versions in CI

File: setup.py [hunk 3]
```diff
@@ -37,7 +37,7 @@
     "pytest-html==1.22.0",  # Newer versions require Python 3
     "pytest-cov==2.7.1",
     "pytest-helpers-namespace==2019.1.8",
```
Comment (oyvindio) [inline setup.py:39]: It may be worth checking whether it is possible to update the pytest plugins together with pytest itself

File: tox.ini (hunk 2 above)
Comment (blockjesper) [reply tox.ini]: Good idea. Fixed.

File: docs/requirements.in (hunk 1 above)
Comment (blockjesper) [reply docs/requirements.in:6]: Fixed!

File: setup.py (hunk 3 above)
Comment (blockjesper) [reply setup.py]: Fixed

**Pull request merged**